experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -71,9 +71,14 @@ end
 # extensions required by the Interfacer
 get_field(sim::ClimaAtmosSimulation, ::Val{:radiative_energy_flux}) =
     Fields.level(sim.integrator.p.ᶠradiation_flux, half)
-get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation}) =
-    sim.integrator.p.col_integrated_rain .+ sim.integrator.p.col_integrated_snow # all fallen snow melts for now
-get_field(sim::ClimaAtmosSimulation, ::Val{:snow_precipitation}) = sim.integrator.p.col_integrated_snow .* FT(0)
+function get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation})
+    ρ_liq = CAP.ρ_cloud_liq(sim.integrator.p.params)
+    sim.integrator.p.col_integrated_rain .* ρ_liq # kg/m^2/s
```

Comment:
Will we be able to remove this access to `sim.integrator.p.col_integrated_rain` and generalize it as well at some point in the future?

Reply:
the `..._init.jl` files is what we'd like the component models to adopt, so eventually this will go to ClimaAtmos. I think this won't be until the Atmos interface is modularized and tested though. We definitely don't want our `src/` code to contain any model specific data structures, and this is the last module that contained them! 😅